.coafile

```diff
@@ -0,0 +1,18 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, .eslintrc.js
+
+[all.markdown]
+bears = MarkdownBear
+files = *.md
+
+[all.json]
+bears = JSONFormatBear
+files = *.json
+
```

Comment:
FYI JSONFormatBear can be set to indent at 2 spaces instead of 4, to keep the indent on `package.json` and `bower.json`

I'm not sure if `yarn` and `bower` picks up the indent and use it for doing future modification on the json, I'm worried it might reset it to 2 spaces again.

Or maybe just ignore `package.json` and `bower.json` since that files is mostly modified with the appropriate CLI tool.

Reply:
Maybe it isn't a good idea to ignore it ...

Reply:
I tried yarn, and it adhere the 4 space indent.

Reply:
I tried yarn, and it adhere the 4 space indent.

Reply:
Ok, try bower?

Reply:
Ok where's the ignore config

Reply:
and why not just set the config to indent at 2 spaces? is there another json file that uses 4 spaces?

Reply:
It is the default by JSONLintBear. I don't really know what is the best style. do a basic support for default.

Reply:
My assumsion, the default is the coala prefered, or the most widely used.